Core/Tools/mangler/wlib/streamer.cpp

```diff
@@ -22,15 +22,15 @@
   #include <windows.h>
 #endif
 
-#if defined(USING_STLPORT) || (defined(_MSC_VER) && _MSC_VER < 1300)
```

Comment:
Shouldn't this be `#if defined(USING_STLPORT) || defined(VC6_BUILD)` ? As written it just checks if VC6_BUILD is none zero.